### PR TITLE
demo: build simple calculator in c

### PR DESCRIPTION
**Pull request by DestinedCodes**
This can be used as a start (boilerplate) for the rest of the project.
I added a header file **`calc.h`** which contains the **standard inpu**t and output Library.
I also wrote the **`calc.c`** file which house the **main function** and implements the basic calculator

image

more features coming soon

### CODE REVIEW

**Comment by dohoudaniel** [comment]
Hi Destiny.
Please move these files you wrote (they are awesome!!) into a new directory called Destined_Codes-C. Don't forget to add a README.
Thank you boss.

**Comment by DestinedCodes** [comment]
Sure, I'll do that

**Comment by DestinedCodes** [comment]
@dohoudaniel I have moved the Source code to the `DestinedCodes-c` directory

**Comment by dohoudaniel** [comment]
Thank you.
Reviewers will be assigned soon.
Thank you @DestinedCodes

File: DestinedCodes-c/calc.c [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		else
+		{
+			printf("Error: Division by zero is not allowed.\n");
+			return (1);
+		}
+		break;
+	default:
+		printf("Error: Invalid operator '%c'.\n", operator);
+		return (1);
+	}
```
Comment (DestinedCodes) [inline DestinedCodes-c/calc.c:42]: using if else might be modular in this case compared to using a switch case

**Pull request merged**